python/dgl/nn/pytorch/factory.py

```diff
@@ -113,13 +114,21 @@ def forward(self, x, algorithm='bruteforce-blas', dist='euclidean'):
               :math:`\sqrt{\sum_{i} (x_{i} - y_{i})^{2}}`.
             * 'cosine': Use cosine distance.
             (default: 'euclidean')
+        exclude_self : bool, optional
+            If True, the output graph will not contain self loop edges, and each node will not
+            be counted as one of its own k neighbors.  If False, the output graph will contain
+            self loop edges, and a node will be counted as one of its own k neighbors.
+        output_batch : bool, optional
+            If True, the output graph will have batch information set according to the
+            shape of ``x``. If False, the output graph will not have batch information set.
```

Comment:
"Batch information set according to the shape of x" sounds too vague.  I would prefer something more concrete, like:
"If True, the output will be a batched graph with xxx graphs.  Each graph within will be a KNN graph constructed from ..." (or something similar).

Reply:
Thanks.  I've now updated the comments to be more specific, hopefully without being too verbose, since the multiple KNN graphs part is described elsewhere in the doc comment.  It's described in even more detail in the doc comments in functional.py, so maybe that part of the comment could be brought over here in factory.py, if people would prefer. 
 However, the output graph representing multiple KNN graphs or not is independent of whether the batch info is set or not for those graphs, so people might want that doc change to be separate from this one, or maybe not.